frame_test.go

```diff
@@ -135,3 +134,27 @@ func TestReservedBytes(t *testing.T) {
 		},
 		buf.Bytes(), "Unexpected bytes")
 }
+
+func TestService(t *testing.T) {
+	f := NewFrame(100)
+	fh := fakeHeader()
+	f.Header = fh
+	fh.write(typed.NewWriteBuffer(f.headerBuffer))
+
+	payload := typed.NewWriteBuffer(f.Payload)
+	payload.WriteSingleByte(0)
+	payload.WriteUint32(42)
+	payload.WriteBytes(make([]byte, 25))
+	payload.WriteLen8String("bankmoji")
```

Comment:
instead of creating a payload from scratch, you might want to create a `callReq` and then serialize that to a frame.
e.g.
https://github.com/uber/tchannel-go/blob/dev/init_test.go#L36